### PR TITLE
Feature/accessibility

### PR DESCRIPTION
**Pull request by hjawad22**
### Types of changes made?
- [ ] HTML
### What does this PR do?
- Changes some HTML elements to be more semantic 
### -Questions/Comments/Relevant Screenshots?
 Is there anything I could be doing to the HTML to make it more accessible?

### CODE REVIEW

**Comment by hjawad22** [comment]
@robbiejaeger Second review to my lovely accountability buddy.

File: dist/index.html [hunk 1]
```diff
@@ -1,40 +1,40 @@
...
+               <label for="end" class="label">End Date:</label>
+               <input type="date" id="end" class="style" name="date-end" min="2023-04-25" aria-label="Choose a end date" required>
+               <button class="book-button"><strong>Book Your Next Adventure</strong></button>
+               <button class="quote-button"><strong>View Trip Estimation</strong></button>
+               <p class="estimated-total"></p>
+            </form>
+         </section>
+         <section class="trips-area">
+            <h2>Your Trips</h2>
+            <p class="total-spent-message"><strong></strong></p>
```
Comment (asutherland91) [inline dist/index.html:34]: You may want to consider instead of using a strong HTML element, making it bold in your CSS instead

Comment (asutherland91) [reply dist/index.html:34]: I see this on a few different lines, styling is best done in CSS

**Review by asutherland91 — COMMENTED** [review]
Your HTML looks neat and well formatted as I can see. I would just consider changing your bold styling to being in CSS instead. I left an inline comment on one of the lines I saw it on. Great job, can't wait to see it completed!

**Pull request merged**